### DISCUSSION TITLE
Cannot add computed channel

### DISCUSSION THREAD

**Discussion by tc3sean** [danielhrisca/asammdf#1067] [Q&A]
Hello all, I am having trouble using the computation feature that used to work in a previous version. 

When I press Ins key, I am greeted with their error. Can someone assist me how I can get the computation to work? 

Much appreciated. 

![image](https://github.com/user-attachments/assets/abd6d6ba-5c3b-4fc4-b2ad-bc907ba60d90)

**Comment by danielhrisca** [comment]
See here https://asammdf.readthedocs.io/en/latest/functions_manager.html

**Reply by tc3sean** [reply]
Thank you for the prompt reply about the computation, but before I try that I am now running into a basic issue that I did not have with previous version. When I try to extract bus signals, a processing window comes up but a new MF4 is not created. I have downloaded the v7.5.0d2 from css electronics website.

**Reply by danielhrisca** [reply]
you may try the development build here https://github.com/danielhrisca/asammdf/actions/runs/10503162865
